jvm/src/main/kotlin/app/cash/trifle/delegate/CertificateAuthorityDelegate.kt

```diff
@@ -22,9 +22,16 @@ interface CertificateAuthorityDelegate {
    *
    * @param issuerCertificate trifle certificate associated with the signer of this cert.
    * @param certificateRequest certificate request used to generate a new certificate
+   * @param validity of the certificate before it expires (days)
    */
   fun signCertificate(
     issuerCertificate: Certificate,
     certificateRequest: CertificateRequest,
+    validity: Period = Period.ofDays(MOBILE_CERTIFICATE_VALIDITY_PERIOD_DAYS),
   ): Certificate
+
+  companion object {
+    // Validity time for device-certificate, currently scoped to 1 day
+    internal const val  MOBILE_CERTIFICATE_VALIDITY_PERIOD_DAYS: Int = 1
```

Comment:
do we want to keep the default value to 30 and have apps overwrite this value instead?

Reply:
I wouldnt mind doing this in code (for both prod and staging) while we havent implemented the error propagation from server side.

Reply:
update: setting back to 30 days and changing to 1 in calling code.